tests/test_dataset_generator.py

```diff
@@ -0,0 +1,161 @@
+                    "12/26/2018, 04:34:56",
+                ]
+            ),
+        ]
+        ordered_data = [
+            np.array(
+                [
+                    "February 20 2019 13:00:00",
+                    "February 21 2019 12:00:00",
+                    "February 21 2019 13:00:00",
+                    "March 21 2019 13:00:00",
+                    "February 20 2020 13:00:00",
+                ]
+            ),
+            np.array(
+                [
+                    "12/26/2017, 04:34:52",
+                    "11/26/2018, 04:34:52",
+                    "12/26/2018, 04:34:52",
+                    "12/26/2018, 04:34:56",
+                    "12/27/2018, 04:34:52",
+                ]
+            ),
+        ]
+
+        output_data = []
+        output_data.append(dg.get_ordered_column(data[0], "datetime"))
+        output_data.append(dg.get_ordered_column(data[1], "datetime", date_format))
+
+        for i in range(len(output_data)):
+            self.assertTrue(np.array_equal(output_data[i], ordered_data[i]))
+
+    def test_get_ordered_column_datetime_descending(self):
+        date_format = "%m/%d/%Y, %H:%M:%S"
+        data = [
+            np.array(
+                [
+                    "February 20 2019 13:00:00",
+                    "February 21 2019 12:00:00",
+                    "February 21 2019 13:00:00",
+                    "March 21 2019 13:00:00",
+                    "February 20 2020 13:00:00",
+                ]
+            ),
+            np.array(
+                [
+                    "12/26/2017, 04:34:52",
+                    "11/26/2018, 04:34:52",
+                    "12/26/2018, 04:34:52",
+                    "12/26/2018, 04:34:56",
+                    "12/27/2018, 04:34:52",
+                ]
+            ),
+        ]
+        ordered_data = [
+            np.array(
+                [
+                    "February 20 2020 13:00:00",
+                    "March 21 2019 13:00:00",
+                    "February 21 2019 13:00:00",
+                    "February 21 2019 12:00:00",
+                    "February 20 2019 13:00:00",
+                ]
+            ),
+            np.array(
+                [
+                    "12/27/2018, 04:34:52",
+                    "12/26/2018, 04:34:56",
+                    "12/26/2018, 04:34:52",
+                    "11/26/2018, 04:34:52",
+                    "12/26/2017, 04:34:52",
+                ]
+            ),
+        ]
+        output_data = []
+        output_data.append(
+            dg.get_ordered_column(data[0], "datetime", order="descending")
+        )
+        output_data.append(
+            dg.get_ordered_column(data[1], "datetime", date_format, order="descending")
+        )
+
+        for i in range(len(output_data)):
+            print(output_data[i], "GG", ordered_data[i])
+            self.assertTrue(np.array_equal(output_data[i], ordered_data[i]))
+
+    def test_get_ordered_column(self):
```

Comment:
These last two tests test that get_ordered_column returns numpy array in ascending/descending order